### DISCUSSION TITLE
Train before each use ?

### DISCUSSION THREAD

**Discussion by t-dmac** [kyleskom/NBA-Machine-Learning-Sports-Betting#426] [Q&A]
Hello,

New here, should i train and run these steps before every use for best results? 

# Create dataset with the latest data for 2023-24 season
cd src/Process-Data
python -m Get_Data
python -m Get_Odds_Data
python -m Create_Games

# Train models
cd ../Train-Models
python -m XGBoost_Model_ML
python -m XGBoost_Model_UO

or is this something you do occasionally ? 

Also, could someone share a discord link.

Thanks,

**Comment by HyakuzaO** [comment]
I guess you dont need To train the model for daily predictions, or train once a week. But older guys should give better advices. I talk for me.
